src/test/java/uk/gov/hmcts/reform/migration/service/ApplicantDataMigrationServiceTest.java

```diff
@@ -0,0 +1,229 @@
+package uk.gov.hmcts.reform.migration.service;
+
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.domain.Applicant;
+import uk.gov.hmcts.reform.domain.OldApplicant;
+import uk.gov.hmcts.reform.domain.common.Address;
+import uk.gov.hmcts.reform.domain.common.EmailAddress;
+import uk.gov.hmcts.reform.domain.common.MobileNumber;
+import uk.gov.hmcts.reform.domain.common.ApplicantParty;
+import uk.gov.hmcts.reform.domain.common.TelephoneNumber;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ApplicantDataMigrationServiceTest {
```

Comment:
There is no test for an empty `applicant` object that I guess can happen. Please add.

Reply:
@swalker125 what way would this be tested?

Reply:
👍